conductr_cli/conduct_main.py

```diff
@@ -494,6 +495,19 @@ def build_parser(dcos_mode):
 
     add_default_arguments(backup_parser, dcos_mode)
     backup_parser.set_defaults(func=conductr_backup.backup)
+
+    restore_parser = subparsers.add_parser('restore',
+                                           help='Restore a cluster from backup.',
+                                           formatter_class=argparse.RawTextHelpFormatter)
+
+    restore_parser.add_argument('-backup',
```

Comment:
What about no name, i.e. just `conduct restore [path-to-backup]` instead of `conduct restore -backup [path-to-backup]`